streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1361,6 +1368,11 @@ public void start(Application appParam, boolean auto) throws Exception {
         } else {
             if (ExecutionMode.isKubernetesApplicationMode(application.getExecutionMode())) {
                 AssertUtils.state(buildResult != null);
+                String flinkConfPath = String.format("%s/conf/flink-conf.yaml", flinkEnv.getFlinkHome());
+                ParameterTool parameter = ParameterTool.fromPropertiesFile(flinkConfPath);
```

Comment:
Just check `flink-conf.yaml`? Should we check properties as well?

Reply:
properties？ flink-conf.properties file?

Reply:
String fsPath = Optional.ofNullable(parameter.get(JobManagerOptions.ARCHIVE_DIR.key())).orElse(archivePath);
                
How to flink-conf.yaml configuration no, will go to the default ：orElse(archivePath)

Reply:
If user set the `JobManagerOptions.ARCHIVE_DIR.key()` in the `-D` properties, should we check it?